aiohttp/helpers.py

```diff
@@ -50,17 +50,22 @@
 from urllib.parse import quote
 from urllib.request import getproxies, proxy_bypass
 
-import async_timeout
 from multidict import CIMultiDict, MultiDict, MultiDictProxy
 from yarl import URL
 
 from . import hdrs
 from .log import client_logger
 from .typedefs import PathLike  # noqa
 
+if sys.version_info >= (3, 11):
+    import asyncio as async_timeout
+else:
+    import async_timeout
+
 __all__ = ("BasicAuth", "ChainMapProxy", "ETag")
 
 PY_310 = sys.version_info >= (3, 10)
+PY_311 = sys.version_info >= (3, 11)
```

Comment:
Personally, I'm in favour of removing these variables (or atleast not adding new ones). mypy doesn't seem to work well with them, and they seem to actually end up being more work for me when I come to remove them after dropping support for a given Python version.

Reply:
Works for me! Yeah, mypy was the reason I used `sys.version_info` for the import guard already. Just wasn't sure which you prefer.